### PR TITLE
Add instance tags.

### PR DESCRIPTION
**Pull request by powellchristoph**

### CODE REVIEW

File: modules/node/variables.tf [hunk 1]
```diff
@@ -23,6 +23,11 @@ variable "image_name" {
   type = string
 }
 
+variable "instance_tags" {
+  type    = list(any)
```
Comment (remche) [inline modules/node/variables.tf:27]: This should be `list(string)` according to the provider doc ;)

File: variables.tf [hunk 2]
```diff
@@ -101,6 +101,12 @@ variable "image_name" {
   description = "ID of image nodes (must fullfill [RKE2 requirements](https://docs.rke2.io/install/requirements/))"
 }
 
+variable "instance_tags" {
+  type        = list(any)
```
Comment (remche) [inline variables.tf:105]: This should be `list(string)` according to the provider doc ;)

**Comment by remche** [comment]
thanks for your PR, please see the comments :)

**Comment by powellchristoph** [comment]
@remche I don't suppose you could cut a new release? Thanks!

**Comment by remche** [comment]
@powellchristoph done !

**Pull request merged**